controller/persistence/identity_store.go

```diff
@@ -26,14 +26,13 @@ import (
 )
 
 const (
-	FieldIdentityType             = "type"
-	FieldIdentityAppwans          = "appwans"
-	FieldIdentityApiSessions      = "apiSessions"
-	FieldIdentityHostableServices = "hostableServices"
-	FieldIdentityIsDefaultAdmin   = "isDefaultAdmin"
-	FieldIdentityIsAdmin          = "isAdmin"
-	FieldIdentityEnrollments      = "enrollments"
-	FieldIdentityAuthenticators   = "authenticators"
+	FieldIdentityType           = "type"
+	FieldIdentityAppwans        = "appwans"
```

Comment:
I assume these appwans defs need to stay for migration purposes?

Reply:
Yes, for now. Going to strip everything out at once.